src/app/shared/http-services/activity-log.service.ts

```diff
@@ -47,6 +47,12 @@ export interface ActivityLog {
     type: 'Team'|'User',
   },
   score?: number,
+  user?: {
+    id: string,
+    first_name: string|null,
+    last_name: string|null,
```

Comment:
first/lastnames are marked as not "required" in the API, that means that they may be undefined as well!

Reply:
Fixed